superset-frontend/src/features/rls/RowLevelSecurityModal.tsx

```diff
@@ -39,6 +39,7 @@ import { FilterType, RLSObject, RoleObject, TableObject } from './types';
 
 const StyledModal = styled(Modal)`
   max-width: 1200px;
+  min-width: 352px;
```

Comment:
I would recommend so that you don't need to worry about the actual width of this modal changing, if say the global styles of the buttons change at some point, try using `min-width: min-content` here and then for the footer, block it from wrapping with `white-space: nowrap;`